base.mk

```diff
@@ -28,11 +28,11 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
 	ro.build.version.security_patch=$(PLATFORM_SECURITY_PATCH) \
 	ro.adb.secure=0 \
 	ro.logd.auditd=true
-	
```

Comment:
uh looks like github lost my comment.
Please remove this

Reply:
Done